core/src/main/scala/dagr/core/cmdline/DagrCoreMain.scala

```diff
@@ -189,10 +186,20 @@ class DagrCoreMain(
       this.reportPath = pick(report, pipeline.outputDirectory.map(_.resolve("execution_report.txt")), Option(Io.StdOut)).map(_.toAbsolutePath)
       this.reportPath.foreach(p => Io.assertCanWriteFile(p, parentMustExist=false))
 
-      Logger.level = this.logLevel
+      if (0 < taskTop && !Terminal.supportsAnsi) throw new ValidationException("ANSI codes are not supported in your terminal.  Interactive mode will not function properly.")
+
+      // TODO: Perhaps have the top-like interface have a line or two that shows the last info message?
+      Logger.level = if (0 < taskTop) LogLevel.Error else this.logLevel
 
       val resources = TaskManagerResources(cores = cores.map(Cores(_)), totalMemory = memory.map(Memory(_)))
-      this.taskManager = Some(new TaskManager(taskManagerResources=resources, scriptsDirectory = scriptsDirectory, logDirectory = logDirectory))
+      this.taskManager = Some(
+        new TaskManager(
+          taskManagerResources=resources,
+          scriptsDirectory = scriptsDirectory,
+          logDirectory = logDirectory,
+          taskTop = taskTop
```

Comment:
This surprised me a bit.  I would have thought it would be good to keep this out of task manager, and instead have the top-like code be in or called from DagrMain.